ReactiveCocoa/Reusable.swift

```diff
@@ -0,0 +1,10 @@
+import ReactiveSwift
+import enum Result.NoError
+
+public protocol Reusable: class {}
```

Comment:
As said the compiler complains about the conformance of `NSCollectionViewItem`'s `prepareForReuse` to `Resuable`.

<img width="722" alt="screen shot 2016-10-17 at 11 59 19 pm" src="https://cloud.githubusercontent.com/assets/11806295/19457249/bf5481be-94c5-11e6-985e-4be738a23ec6.png">